java/maven/src/org/netbeans/modules/maven/ActionProviderImpl.java

```diff
@@ -304,9 +309,24 @@ private boolean checkSurefire(final String action) {
                         RequestProcessor.getDefault().post(new Runnable() {
                             @Override
                             public void run() {
+                                String surefireVersion = null;
+                                String junitVersion = null;
+                                
+                                if (ju5 && !usingSurefire2_22()) {
+                                    surefireVersion = SUREFIRE_VERSION_SAFE_5;
+                                } else if (usingSurefire28()) {
```

Comment:
Not sure about the logic here - should it be:
```suggestion
                                } else if (!usingSurefire28()) {
```

Reply:
Typo - thanks !

Reply:
Works much better ... in 78778ee